tests/e2e/configurer/chain/commands.go

```diff
@@ -118,7 +118,7 @@ func (n *NodeConfig) StoreWasmCode(wasmFile, from string) {
 
 func (n *NodeConfig) WithdrawPosition(from, liquidityOut string, positionId uint64) {
 	n.LogActionF("withdrawing liquidity from position")
-	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "withdraw-position", fmt.Sprint(positionId), liquidityOut, fmt.Sprintf("--from=%s", from), "--gas=700000", "--fees=5000uosmo"}
+	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "withdraw-position", fmt.Sprint(positionId), liquidityOut, fmt.Sprintf("--from=%s", from), "--gas=800000", "--fees=5000uosmo"}
```

Comment:
I think we were previously calculating a zero value for the forfeited amount that should be sent to the community pool. Since E2E tests the first and only position and withdraws that position, we now are actually doing an extra send to the community pool, which increases the gas value here